toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -84,7 +84,7 @@ script_dir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
 topdir=/usr/src/azl
 enable_local_repo=false
 keep_container="--rm"
-packages_to_install="azurelinux-release vim git jq"
+packages_to_install="azurelinux-release vim git jq azurelinux-release"
```

Comment:
Is this a typo or an indented change.
I can see the first instance to be `azurelinux-release`